### DISCUSSION TITLE
idea to quicken the learning curve. something i struggled with that others shouldn't have to.

### DISCUSSION THREAD

**Discussion by somethingstarted** [microsoft/vcpkg#28785] [Ideas]
I think a quick sentence or two on every page* where a vcpkg is should have a description or example on how to plug it in. it took me 3 days of searching and googling just to figure out you do .\vcpkg LibraryYouInstall:Compatability-Version-You-Install. 

maybe on the web page put ".\vcpkg install 7zip:[compatibility version]" or something, or a little thing to encourage you to click on a compatibility version and then it updates the ".\vcpkg install 7zip" to say, say ".\vcpkg install 7zip:x64-linux" or whatever, then if you click it again it removes it from that little text thingy. or maybe it could have a checkmark next to it and you click all the ones you want installed, and then it rebuilds the installation text to get it to automagically do one after the other with the manifest feature. I think that would GREATLY improve ease of use and navigability 

*talking about these pages: https://vcpkg.io/en/packages.html

this idea would be especially helpful for those who are self-taught folks who don't have anyone around them who've used it yet, but only seen a few videos from a few youtubers on it or some reddit post

**Comment by somethingstarted** [comment]
also, on the program itself, if you just type in, say ".\vcpkg install wxwidgets" it should then first check for other versions of it, and say 
"here are the compatability versions of wxwidgets, which ones would you like installed?

0. cancel, i changed my mind. 
1. all versions below:
2. this version
3.  this different version
4. this other different version"  

and then a simple way of picking  which ones, maybe answer with a "1" or a "2, 4" and it would know to install just #2 and 4. 

anyways... i think this would make it so you only really have to learn about triplets and stuff if you start getting deep. many of the "triplet" talk can be abstracted into just saying ":compatability-name" - and then there could be some logic to just know to instal whatever it takes to make triplets work "installing things to make triplets / compatibility types work, should only need to do this once per whatever"

**Reply by Osyotr** [reply]
> many of the "triplet" talk can be abstracted into just saying ":compatability-name"

Please don't invent new names for things that already have perfectly reasonable names and documentation.
